lisp/core/install-vc.el

```diff
@@ -0,0 +1,71 @@
+;;; core/install-vc.el --- Install packages directly from the version control  -*- lexical-binding: t; -*-
+
+;;; Commentary:
+;;
+;; Command use to install packages directly from the version control
+;;
+;;   $ eask install-vc [specs..]
+;;
+;;
+;;  Positionals:
+;;
+;;    [specs..]     specs to install as packages; it will install through the
+;;                  function `package-vc-install'
+;;
+
+;;; Code:
+
+(let ((dir (file-name-directory (nth 1 (member "-scriptload" command-line-args)))))
+  (load (expand-file-name "_prepare.el"
+                          (locate-dominating-file dir "_prepare.el"))
+        nil t))
+
+(eask-load "core/install-file")
+
+(defun eask-install-vc--split-sepcs (specs)
```

Comment:
Doesn't work for e.g. `company-mode/company-mode` where the package is named `company`

Reply:
Yeah, this probably doesn't work. 

This is a rough implementation since I currently doesn't have a good idea to handle multiple specs. 🤔 But I'm looking forward to improve this!